action/protocol/rewarding/reward.go

```diff
@@ -365,10 +365,10 @@ func (p *Protocol) grantToAccount(ctx context.Context, sm protocol.StateManager,
 	return p.putState(ctx, sm, accKey, &acc)
 }
 
-func (p *Protocol) claimFromAccount(ctx context.Context, sm protocol.StateManager, addr address.Address, amount *big.Int) error {
+func (p *Protocol) claimFromAccount(ctx context.Context, sm protocol.StateManager, from, to address.Address, amount *big.Int) error {
 	// Update reward account
 	acc := rewardAccount{}
-	accKey := append(_adminKey, addr.Bytes()...)
+	accKey := append(_adminKey, from.Bytes()...)
```

Comment:
the current code used `claimTo` from L288, there's no balance in this address (if different from `actionCtx.Caller`)